staging/src/k8s.io/noderesourcetopology-api/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - swatisehgal
+  - fromanirh
+reviewers:
+  - swatisehgal
+  - fromanirh
+  - AlexeyPerevalov
```

Comment:
I would suggest to have @marquiz  added here as well as approver and/or reviewer. It would be good to have NFD maintainer to closely following those.

Reply:
Sure, will add @marquiz here.